test/cdmparse.jl

```diff
@@ -27,7 +27,7 @@ end
 
 @testset "CDM parsing invalid directory" begin
     baddir = joinpath(RTS_GMLC_DIR, "../../test")
-    @test_throws SystemError PowerSystems.csv2ps_dict(baddir, 100.0)
+    @test_throws ErrorException PowerSystems.csv2ps_dict(baddir, 100.0)
```

Comment:
This throws a `SystemError` on my machine. Am I doing something wrong?

Reply:
It throws ErrorException on mine!  I thought this was a merge problem.  I'll investigate.

Reply:
I pushed a fix.